### ISSUE TITLE
Multiple accounts same network IP

### ISSUE THREAD

**Issue by jerryforcode** [InstaPy/InstaPy#4477]
Hello,

I plan on using multiple raspberry pis to run multiple instagram accounts at different times. How many Instagram accounts can I use per raspberry pi given that it will be on the same network? Is this possible ?

I am assuming Instagram will be aware of multiple accounts running on the same network regardless of how many raspberry pi’s I use...is there work arounds? 

Any guidance is appreciated.

Thank you

**Comment by fraress** [comment]
i think i've read somewhere that instagram start blocking you if it found 5 account working with the same ip. you might use some vpn...does it makes sense?

**Comment by jerryforcode** [comment]
Yes that makes sense. Thank you @fraress

**Comment by jerryforcode** [comment]
@fraress Do you have any recommendations for VPN products?

**Comment by fintzd** [comment]
> i think i've read somewhere that instagram start blocking you if it found 5 account working with the same ip. you might use some vpn...does it makes sense?

Don't use a VPN (such as NordVPN) as the IPs there are used by tons of users so you will get blocked. Either setup a Proxy or buy one, then you can use like 2 accounts per proxy.

**Comment by fraress** [comment]
i m actualy using an instagram bot (bought from a site,now i m looking for making one by myself) and i m running 5 account at the same time, and it s working..sometimes i takes some bans (a couple of day or a week) because i m making to much activity with one of them..but 5 account with the same ip work!

**Comment by fintzd** [comment]
> i m actualy using an instagram bot (bought from a site,now i m looking for making one by myself) and i m running 5 account at the same time, and it s working..sometimes i takes some bans (a couple of day or a week) because i m making to much activity with one of them..but 5 account with the same ip work!

Which one did you buy / why do you want to build one? Is it not worth it? 
And yes, I mean the bans etc depend on the amount of activity you set!

**Comment by fraress** [comment]
> Which one did you buy / why do you want to build one? Is it not worth it?
And yes, I mean the bans etc depend on the amount of activity you set!
> 

i bought followliker with the possibility to run 5 accounts (i paid only a tantum.. now if you are looking to buy it you will have an initial price and other payments every moth..it s quite expensive)
the issuses with this bot is that i have only 5 accounts, it s not that easy to set ( a lot of non intuitive options) and sometimes it messes my account...like loosing all my friends because of the unfollowing tasks (so i have to refollow all of my friend with the usual dm "why were you not following me?!?!"...it s a bit frustrating!) or, this bot do some follow activity based on some hastags or following others people's followers AND some liking activity based on some hastags..its a wrong way to do that!. It would be better (in terms of followers gains) if i can crave people, put a follow some likes and THEN move on on the next person..my bot take a person put a follow, find another person puts some likes...it s not the most efficient way to crave followers!

I would like to create my bot, because i want to play more with the tasks and the timing of the task.
for example:
-starting from hashtags, specific people's followers, and locations (i m an track and field athlete, it will be interesting to find every time the competitions from the federation website..and looks for people on that place..local, national or internation meeting..there is an olympic game, pam! my bot will look for people in that place..there is a national championship? pam! covered! its more efficient -at least for me- to gain followers..same group of people same interest) i put a follow AND 3-4 likes for that person and THEN i move on to the next person. 
-the more you use a bot the more you can follow or liking..for a new account the first week with just 200-300 people followed you may take a ban after years of using you can easly pass 1000 people. So you have to take it slowly and over the weeks/month you can increase this activity. i would like that the bot do that! not me manually changing the options!
-and again if i follow 200 people a day i want to use 24 hours to do it. Not 6 hours..(on followliker you can set a delay time for every activity..but you don't know if it will takes 2-3-8-9-13 hours with your settings to complete it!) the bot may calculate the times that it need to complete everythings and setting the best delay option..even better..it may simulate a normal person, working in the mornig take some pause working again.. (there has to be a minum delay or instagram will give you a ban of course, but it s something that can be implemented)
-it will be even more interesting developing an AI that can find your best way, for you.  to crave followers! (some ideas...find interest categorys like sport, art, fashion...for each of them find specific hashtags and famous account correlate with that particular interest..and then run some follow-liking test...10 follows with that hashtag on male's accounts same but on female's accounts..after this test where i gain the most followers? we can play on age, following/followers of the person, hastag, number of likes....there is a lot of data that can be use! after all of this tests..pem! the bot knows where to look!:
this are the best hashtags for your account (the first 15 hashtags chosen from 1000 hashtags!) 
pam! this is the kind of people you have to look for ! MIND BLOWING
-the bot also can perform some "keep the followers" and it consist on putting some likes on your followers...it going to be something like "hey.. hey! don't sleep on me man! i m here! remember to smash  my photos!" just to be sure that they don't unfollow you, or they forget you!
-other idea, if i follow a person when followliker is running for some strange reason, during an unfollow task, it will unfollow that person, god damnit. i want to keep him on my following!.. you bastard. i want to implement this thing!

what you guys think? someone taht have the passion to write this things?! are you there?!?!

(sorry for the english! i haven't use google translate, I am proud of myself, and my English teacher should be too)

**Comment by Tr1pke** [comment]
i wouldn't go more then 2 accounts per IP, if you really want to push it 3 is the max
else you get temp blocked, i have it a lot with 3 accounts on 1 IP

**Comment by ishandutta2007** [comment]
Couple of new changes I found:
- If you are getting back too many follows without posting anything new, instagram suspects something fishy and throws up account for phone verification
- If you are unfollowing a lot more accounts than following you might be blocked for long term or permanently. The account of mine which got blocked indefinitely I had unfollowed 500 in a day and followed about 200 odd in the day. The block has already been more than 2 weeks now, still continuing. The block is only on follows. I am able to like , comment and post.
